core/src/ons/platform.js

```diff
@@ -138,14 +138,17 @@ class Platform {
   //----------------
   /**
    * @method isIOS
-   * @signature isIOS()
+   * @signature isIOS([ignoreSelectedPlatform])
+   * @param {Boolean} ignoreSelectedPlatform
```

Comment:
This is a better implementation than I was originally thinking of, nice one. I think this parameter could be renamed though. How about `forceActualPlatform`?